README.md

```diff
@@ -277,155 +291,137 @@ my_app> Post.byId("373143473418666496")!.replace({ title: "My dog and other marv
-$ fauna --help [COMMAND]
-USAGE
-  $ fauna COMMAND
-...
-```
-
 # Connecting to different endpoints
 
-We can add endpoints by calling the following command `add-endpoint`. We will be prompted to enter the authentication key and an alias for the endpoint.
+We can add endpoints by calling the following command `endpoint add`. This is
+meant to be used when connecting to a docker container.
 
-```sh-session
-$ fauna add-endpoint "https://example.com"
-Endpoint Key: ****************************************
-Endpoint Alias [example.com]: example_alias
+```sh
+$ fauna endpoint add
+? Endpoint name localhost
+? Database URL http://localhost:8443
+? Database Secret secret
+Checking secret... done
+? Make this endpoint default [no]
+Saved endpoint localhost to ~/.fauna-shell
 ```
 
-The _Endpoint Alias_ should be a name that helps you remember the purpose of this endpoint.
-
-If we have defined many endpoints, we could set one of them as the default one with the `default-endpoint` command:
-
-```sh-session
-$ fauna default-endpoint cloud
-```
+The endpoint name is an arbitrary name that can be used in the `--endpoint` flag
+of other commands. The database URL should typically be `http://localhost:8443`
+or `https://db.fauna.com`, although it can be set to any URL to a fauna
+instance. The database secret is the secret used to authenticate with that
+database.
 
-The _default endpoint_ will be used by the shell to connect to Fauna if the `--endpoint` flag is not set.
+An endpoint can be set as the default, in which case `fauna shell` and
+`fauna eval` will choose that endpoint by default.
 
-Endpoints can be listed with the `list-endpoints` command like this:
+Endpoints can be listed with the `endpoint list` command like this:
 
-```sh-session
-$ fauna list-endpoints
-localhost
-cloud *
-cluster-us-east
+```sh
+$ fauna endpoint list
+  localhost
+* cloud
 ```
 
-There we see that the `cloud` endpoint has a `*` next to its name, meaning that it's the current default one.
+There we see that the `cloud` endpoint has a `*` next to its name, meaning that
+it's the current default one.
 
 Finally, endpoints will be saved to a `~/.fauna-shell` file like this:
 
 ```ini
 default=cloud
 
-[localhost]
-domain=127.0.0.1
-port=8443
-scheme=http
+[endpoint.localhost]
+url=http://localhost:8443
 secret=secret
-graphqlHost=127.0.0.1
-graphqlPort=8084
 
-
-[cloud]
-domain=db.fauna.com
-scheme=https
+[endpoint.cloud]
 secret=FAUNA_SECRET_KEY
-graphqlHost=graphql.fauna.com
-graphqlPort=443
-
-[cluster-us-east]
-domain=cluster-us-east.example.com
-port=443
-scheme=https
-secret=OTHER_FAUNA_SECRET
-graphqlHost=cluster-us-east.example.com
-graphqlPort=443
 ```
 
 # Connecting to local endpoints
 
-If you are running Fauna locally using our Docker images, you may need to configure the Shell to work with local endpoints so you can interact with the databases running in the Docker containers.
+If you are running Fauna locally using our Docker image, you will need to
```

Comment:
Seems like a good place to add a link to https://docs.fauna.com/fauna/current/tools/dev